apps/web/src/components/Search/SearchBar.tsx

```diff
@@ -1,19 +1,29 @@
 "use client"
 
-import { useState, useEffect, useRef } from "react"
+import { SearchResult } from "@/types/utils"
+import { search } from "@/utils/api"
 import { Dialog, Transition } from "@headlessui/react"
-import { redirect } from "next/navigation"
-import { LuSearch } from "react-icons/lu"
 import { Button } from "@mav/ui/components/buttons"
 import { InputField } from "@mav/ui/components/fields"
+import { redirect } from "next/navigation"
+import { useEffect, useRef, useState } from "react"
+import { LuSearch } from "react-icons/lu"
 import { SearchSection } from "./Section"
-import { search } from "@/utils/api"
-import { SearchResult } from "@/types/utils"
 
-export default function SearchBar({ recentSearches = [], characters = [] }: { recentSearches?: string[], characters?: { name: string; image: string }[] }) {
+export default function SearchBar({
+  recentSearches = [],
+  characters = []
+}: {
+  recentSearches?: string[]
+  characters?: { name: string; image: string }[]
+}) {
   const [isOpen, setIsOpen] = useState(false)
   const [loading, setLoading] = useState(false)
-  const [results, setResults] = useState<SearchResult>({ user: [], artwork: [], character: [] })
+  const [results, setResults] = useState<SearchResult>({
+    user: [],
+    artwork: [],
+    character: []
+  })
   const [query, setQuery] = useState("")
```

Comment:
This would be more clearer, because it sounds like a backend operation

```suggestion
  const [searchQuery, setSearchQuery] = useState("")
```